rustls/Cargo.toml

```diff
@@ -81,9 +81,9 @@ log = { workspace = true, optional = true }
 # in the error result from `crypto::CryptoProvider::set_default()`, which does not seem
 # worth using the `critical-section` feature with its no-std installation requirements.
 # TODO: specify `once_cell` version in single place - as RECENTLY UPDATED in upstream RUSTLS
-# (It may be best to specify the `once_cell` features here, as may be needed for updates moving forward.)
-once_cell = { version = "1.20.3", default-features = false, features = ["alloc", "race"] }
-# EXPLICIT `portable-atomic` dependency REQUIRED for `unsafe-assume-single-core` feature
+# (keeping `once_cell` features list here, with UPDATED FEATURES as needed by this main crate)
+once_cell = { version = "1.20.3", default-features = false, features = ["alloc", "portable-atomic", "race"] }
+# EXPLICIT `portable-atomic` dependency REQUIRED BY BOTH  `critical-section` & `unsafe-assume-single-core` features
 portable-atomic = { version = "1.11", default-features = false, optional = true }
 # TODO: KEEP `portable-atomic-util` OPTIONAL WITH FEATURE CONFIG IN THIS FORK
 portable-atomic-util = { version = "0.2.4", default-features = false, features = ["alloc"] }
```

Comment:
I think a note is needed here that using Arc from `portable-atomic-util` requires a cfg option, as I have been documenting.

In case something else may be used in place of `portable-atomic-util`, then the cfg option may not belong in the documentation. This kind of note could be really helpful.